doc/requests.txt

```diff
@@ -142,6 +145,24 @@ List of Detachment System Events (TC = 0x11)
     0x11  0x00  0x01  yes      latch status                     log
 
 
+List of Input System Requests (TC = 0x15)
+--------------------------------------------------------------------------------
+
+     CID   IID   PRI  Payload  Response  Name/Function          Source
+    ------------------------------------------------------------------------
+    0x01  any   0x02  yes      no        output report          log
+    0x02  any   0x02  yes      no        set feature report     log
+    0x03  any   0x02  yes      yes       get feature report     log
+    0x04  #ID   0x02  yes      yes       subdevice meta (HID)   log
+
+
+List of Input System Events (TC = 0x15)
+--------------------------------------------------------------------------------
+
+     CID   IID   PRI  Payload  Name/Function                    Source
+    ------------------------------------------------------------------------
+    0x00  0x03  0x02  yes      input report (touchpad)          log
```

Comment:
This should probably be `#ID` for IID? With elaboration that keyboard events are sent over IID 0